netlogo-gui/src/main/theme/InterfaceColors.scala

```diff
@@ -246,6 +248,28 @@ object InterfaceColors {
 
   def GRAPHICS_BACKGROUND = Constants.ViewBackground
 
+  def ANNOUNCEMENTS_BANNER_BACKGROUND =
+    theme match {
+      case "classic" => CLASSIC_LIGHT_GREEN
```

Comment:
I believe that the classic theme is only intended to change the widgets, but the interface should be identical to the light theme.